cluster/repository/cluster_repository.go

```diff
@@ -211,7 +211,10 @@ func (m *GormClusterRepository) update(ctx context.Context, existing, c *Cluster
 	if err != nil {
 		return errs.WithStack(err)
 	}
-	err = m.db.Model(existing).Updates(c).Error
+	c.ClusterID = existing.ClusterID
+	c.CreatedAt = existing.CreatedAt
+	c.UpdatedAt = existing.UpdatedAt
```

Comment:
I believe that we don't need to set these params, do we?

Reply:
I think that the `clusterID` is already set - it's used a few lines before: https://github.com/fabric8-services/fabric8-cluster/pull/82/files#diff-6f8a71168444c01769c90f0afe9f4c55R197

Reply:
but, this is just a detail ;-)

Reply:
Yes. It’s set in Save(). But update() is also used by CreateOrSave() where ID is not available.

Reply:
ok, I missed this usage